### ISSUE TITLE
[BUG] - Navbar Menu - Scroll locked on landing when disableAnimation is set to TRUE

### ISSUE THREAD

**Issue by alexanderpehlivanov** [nextui-org/nextui#2272]
### NextUI Version

2.2.9

### Describe the bug

On disableAnimation set to TRUE, the scroll event is blocked on landing. It appears back if you click the toggle icon.

### Steps to Reproduce the Bug or Issue

1. Install NextJS v14.0.4 with app directory, typescript and tailwindcss.
2. Install NextUI v2.2.9 and framer-motion v10.17.0.
3. Create a `Navbar ` component with an additional `Navbar Menu`.
4. Add `disableAnimation` to `Navbar` props.
5. Add a widget bigger than `window.height`, so you can scroll.
6. Run `npm run dev`.
7. Wait for the page to compile successfully.
8. Open the page in the mobile version.
9. After loading the `NavbarMenu` component, it will freeze the page scroll until you toggle it.

### Expected behavior

Scroll unlocked

### Operating System Version

Windows 11

### Browser

Chrome

**Comment by dkudrin** [comment]
+1

**Comment by wingkwong** [comment]
Closing in favor of v3. If the issue is reproducible in v3, please create a new one with a minimal reproducible example.
